src/content/en/tools/workbox/get-started/_shared/config.md

```diff
@@ -0,0 +1,19 @@
+* `globDirectory` is where Workbox watches for changes. `globPatterns`
```

Comment:
Since this is an include file, you need to add the `{# wf_md_include #}` tag to prevent the test suite from freaking out.